code/node-server/index.js

```diff
@@ -6,59 +6,104 @@ const express = require('express')
 const config = require('./config')
 
 const validator = require('./src/middleware/validator.js')
-const logger = require('./src/middleware/logger.js')
+// const logger = require('./src/middleware/logger.js')
```

Comment:
logger is currently broken. Commented it out after merging master in2 this branch